index.js

```diff
@@ -1,15 +0,0 @@
-const role = process.env.ROLE || "client";
-
-if (role === "client") {
-  const express = require("express");
-  const app = express();
-  const port = process.env.PORT || "8080";
-
-  app.use(express.static("dist"));
-
-  app.listen(port, () => {
-    console.log(`Herbert ${role} listing at ${port}`);
-  });
-} else {
-  require("./server");
-}
```

Comment:
This didn't seem to be in use anywhere. Left over from pre-vue.js build ?

Reply:
I use this to run the server code on Heroku. Probably better handled in the Procfile?

Reply:
Ahhh gotcha, ok I'll restore this file and update it to work with the new dist dir layout. We can figure out the Procfile approach later.

Reply:
Updated